components/tags-filter/TagsFilter.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import classnames from 'classnames';
+import FilterBtn from './FilterBtn';
+
+export default function TagsFilter({
+  categorisList,
+  selectedCategories,
+  filterOnChange,
+  pageTitle,
+}) {
+  const disabledBtn = (btn) => {
+    btn.classList.add('-disabled');
+    setTimeout(() => btn.classList.remove('-disabled'), 500);
+  };
+
+  const buttonClick = (e, category) => {
+    const selectedItems = selectedCategories.slice();
+    const position = selectedItems.indexOf(category);
+
+    if (position === -1) {
+      selectedItems.push(category);
+    } else {
+      if (selectedItems.length === 1) {
```

Comment:
@soloviovmax , @antonmorya , could it be `else if(..)` instead of `else { if(..) {...} }`.

Reply:
Agree. But `disabledBtn` call should be removed according to previous comment.

Reply:
Done